app/views/planning_application/permitted_development_rights/_immunity_details.erb

```diff
@@ -7,17 +7,26 @@
     <strong>Note: application may be immune from enforcement</strong>
   </p>
 
-  <h3 class="govuk-heading-s govuk-!-margin-top-5 govuk-!-margin-bottom-1">Applicant information</h3>
+  <%= render(PlanningApplications::ImmunityDetailsComponent.new(immunity_details: @planning_application.immune_proposal_details)) %>
+
+  <h3 class="govuk-heading-s govuk-!-margin-top-5 govuk-!-margin-bottom-1">Assessment summary — Evidence of immunity</h3>
   <ul class="govuk-body immunity-details-sub-list">
-    <% @planning_application.immune_proposal_details.each do |proposal_detail| %>
-      <li>
-        <%= proposal_detail.question %> 
-        <% proposal_detail.response_values.each do |response_value| %>
-          <strong>
-            <%= response_value.to_time.nil? ? response_value : response_value.to_time.to_formatted_s(:day_month_year_slashes) %>
-          </strong>
-        <% end %>
-      </li>
-    <% end %>
+    <li><strong>Evidence cover:</strong>
+      <%= planning_application.immunity_detail.start_date.to_fs(:day_month_year_slashes) %> to
+      <%= planning_application.immunity_detail.end_date.to_fs(:day_month_year_slashes) %>
+     </li>
+    <li><strong>Missing evidence (gap in time):</strong> <%= planning_application.immunity_detail.evidence_gaps? ? "Yes" : "No" %> </li>
   </ul>
+
+  <div class="govuk-body">
+    <hr>
+    <% for evidence_group in @evidence_groups %>
+    <h3>
+      <%= link_to sanitize("#{evidence_group.name} (#{evidence_group.documents.count})<br>#{evidence_group.date_range}"),
+                  new_planning_application_immunity_detail_path(@planning_application) + "#accordion-default-heading-#{evidence_group.id}",
```

Comment:
Another thing I noticed was this links to the section, but doesn't open the accordion. I think it's supposed to be open - what I think Raffaella meant by "link to page/open tab"

Reply:
I'll try to dig into it and see if we can open the accordions by default.